static/app/views/stories/storyTypes.tsx

```diff
@@ -316,9 +316,27 @@ function propSort(a: PropTreeNode, b: PropTreeNode) {
     }
   }
 
-  const nameA = 'name' in a.prop ? a.prop.name : a.prop.definitionFilePath;
-  const nameB = 'name' in b.prop ? b.prop.name : b.prop.definitionFilePath;
-  return nameA.localeCompare(nameB);
+  if ('definitionFilePath' in a.prop && 'definitionFilePath' in b.prop) {
```

Comment:
I fixed the sorting while I was at it as I noticed that it moves the required type to the end instead of start